### DISCUSSION TITLE
Tab bar above bufferline

### DISCUSSION THREAD

**Discussion by hubertkuoch** [ayamir/nvimdots#493] [Q&A]
Hi - nice distro! Was just wondering how did you get the tab bar above nvim bufferline that is shown in the screenshots? Like the lhs and rhs minimalist indicators but i cannot find any plugin in your config that set them. Thanks.

**Comment by charliie-dev** [comment]
U mean [lspsaga outline](https://github.com/glepnir/lspsaga.nvim#lspsaga-outline)?

**Reply by hubertkuoch** [reply]
I mean actually the "1 nvim" lite orange-ish marker on the top lhs of each screenshot above the buffet line. Is this part of terminal emulator?

**Reply by charliie-dev** [reply]
Aha, got it. I think that's part of `wezterm`. Try https://github.com/ayamir/dotfiles/tree/master/catppuccin/.config/wezterm

**Reply by ayamir** [reply]
> I mean actually the "1 nvim" lite orange-ish marker on the top lhs of each screenshot above the buffet line. Is this part of terminal emulator?

It's tmux of https://github.com/catppuccin/tmux

**Reply by hubertkuoch** [reply]
Thanks, yes just spotted it as well. Thanks again for checking.
